src/tests/simulation/src/nexmark.rs

```diff
@@ -37,11 +37,14 @@ impl NexmarkCluster {
         conf: Configuration,
         split_num: usize,
         event_num: Option<usize>,
+        watermark: bool,
```

Comment:
Watermark operators do not survive the scaling test, so I've made this always `false` now. Will investigate it later.